notebooks/122-quantizing-model-with-accuracy-control/122-speech-recognition-quantization-wav2vec2.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p> &gt;<strong>NOTE</strong>: Currently, 8-bit quantization with accuracy control is available only for models in OpenVINO representation. </p><p> </p><p>sounds like you plan other use cases, maybe need to add that it is related to NNCF?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1234/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added